tox.ini

```diff
@@ -7,7 +7,7 @@ skipsdist = True
 passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 usedevelop = True
 deps = -r{toxinidir}/requirements.tests.txt
-commands = py.test -n auto --cov=talisker
+commands = py.test --cov=talisker
```

Comment:
Double "huh?"

Reply:
See above. Travis is already more susceptible to these weird test hangs.